.werft/platform-delete-preview-environments-cron.ts

```diff
@@ -37,48 +37,102 @@ async function deletePreviewEnvironments() {
     }
     werft.done("prep")
 
-    werft.phase("Fetching branches");
-    const branches = getAllBranches();
-    // During the transition from the old preview names to the new ones we have to check for the existence of both the old or new
-    // preview name patterns before it is safe to delete a namespace.
-    const expectedPreviewEnvironmentNamespaces = new Set(branches.flatMap(branch => [parseBranch(branch), expectedNamespaceFromBranch(branch)]));
-    werft.done("Fetching branches");
-
     werft.phase("Fetching previews");
     let previews: string[]
     try {
         previews = listAllPreviewNamespaces(CORE_DEV_KUBECONFIG_PATH, {});
-        previews.forEach(previewNs => werft.log("Fetching previews", previewNs))
+        previews.forEach(previewNs => werft.log("Fetching preview", previewNs));
+        werft.done("Fetching preview");
     } catch (err) {
-        werft.fail("Fetching previews", err)
+        werft.fail("Fetching preview", err)
     }
-    werft.done("Fetching previews");
+
+    werft.phase("Fetching outdated branches");
+    const branches = getAllBranches();
+    const outdatedPreviews = new Set(branches
+        .filter(branch => {
+          const lastCommit = exec(`git log ${branch} --since=$(date +%Y-%m-%d -d "5 days ago")`, { silent: true })
```

Comment:
I would prefer to prefix `origin/` to the branch here and then revert your other changes to getAllBranches ☺️

Reply:
I reverted the changes and added the string as a prefix.